GitUI/BranchTreePanel/RepoObjectsTree.Nodes.Submodules.cs

```diff
@@ -91,9 +96,33 @@ protected override string NodeName()
 
             public void Open()
             {
+                if (Info?.Detailed?.RawStatus is not null)
+                {
+                    UICommands.BrowseSetWorkingDir(Info.Path, ObjectId.WorkTreeId, Info.Detailed.RawStatus.OldCommit);
+                    return;
+                }
+
                 UICommands.BrowseSetWorkingDir(Info.Path);
             }
 
+            public void OpenGE()
```

Comment:
As per https://github.com/gitextensions/gitextensions/pull/8738
```suggestion
            public void LaunchGitExtensions()
```